cropper/src/main/java/com/canhub/cropper/CropImage.kt

```diff
@@ -165,8 +169,10 @@ object CropImage {
         return getPickImageChooserIntent(
             context = context,
             title = context.getString(R.string.pick_image_intent_chooser_title),
-            includeDocuments = false,
-            includeCamera = true
+            options = PickImageContractOptions(
+                includeDocuments = false,
+                includeCamera = true
+            )
```

Comment:
Why we don't have include gallery?

Reply:
It's set to true by default on the PickImageContractOptions class, I'm trying to preserve the exact behavior that was there before.